cit-594-plus.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could we add this file to .gitignore?

Reply:
Done. Should I close this pull request and submit a new one?

Reply:
No, you don't need to close the PRs. You can change push to the same branch to update it.

Reply:
I think you have to actually remove the file from commit as well: 
Not 100% sure, but this command might work: `git rm --cached <file>`

Reply:
How about now? I created another commit by pushing it again. Sorry, so bad at git...

Reply:
I don't see a new commit.

Reply:
![image](https://user-images.githubusercontent.com/37563293/132076977-54e6c456-af96-468e-bc5c-a4979ca6fc9c.png)

Reply:
Because the file was added to before the `.gitignore` was updated, the file is still in the commit. I think using `git rm` should remove the file, and then you can push again to commit the removal of the file.

Reply:
Done?

Reply:
Awesome! You're on your way to mastering git.